internal/encoding/base64.go

```diff
@@ -0,0 +1,23 @@
+package encoding
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"strings"
+)
+
+// DecodeBase64OrJSON decodes a JSON string that can optionally be base64 encoded.
+func DecodeBase64OrJSON(in string, out interface{}) error {
+	in = strings.TrimSpace(in)
+
+	// the service account can be base64 encoded
+	if !strings.HasPrefix(in, "{") {
```

Comment:
I believe JSON can also be prefixed with `[`.

Reply:
https://stackoverflow.com/questions/5034444/can-json-start-with#:~:text=From%20what%20I%20can%20read,an%20array%20element%20in%20JSON.

Reply:
Alternatively... https://golang.org/pkg/encoding/json/#Valid